actionview/lib/action_view/helpers/atom_feed_helper.rb

```diff
@@ -140,9 +140,7 @@ def initialize(xml)
           def method_missing(method, *arguments, &block)
             if xhtml_block?(method, arguments)
               @xml.__send__(method, *arguments) do
-                @xml.div(xmlns: "http://www.w3.org/1999/xhtml") do |xhtml|
-                  block.call(xhtml)
-                end
+                @xml.div(xmlns: "http://www.w3.org/1999/xhtml", &block)
```

Comment:
I think that's the only block related change that truly seem useful here.

@fatkodima can you remove all the block related perf cops except this one? I'm not too sure how this one is named.